### PR TITLE
Installing files for iRonCub-Mk3-WALKING-TURBINES and iRonCub-Mk3-WALKING-NO-TURBINES

### PR DESCRIPTION
**Pull request by davidegorbani**
I modified the `CMakeLists.txt` file to install the files also when the `YARP_ROBOT_NAME=iRonCub-Mk3-WALKING-TURBINES`

### CODE REVIEW

File: iRonCub-Mk3/CMakeLists.txt [hunk 1]
```diff
@@ -7,6 +7,7 @@ option(INSTALL_ALL_IRONCUB_VERSIONS "Install all variations of iRonCub robots."
 
 if(INSTALL_ALL_IRONCUB_VERSIONS)
     set(robot_list iRonCub-Mk3 iRonCub-Mk3-FT-ARMS) #Add to the list other iRonCub3 versions (that share the same configuration files)
+    set(robot_list iRonCub-Mk3 iRonCub-Mk3-WALKING-TURBINES) #Add to the list other iRonCub3 versions (that share the same configuration files)
```
Comment (HosameldinMohamed) [inline iRonCub-Mk3/CMakeLists.txt]: Why didn't you append the list in the lin on top?

Not sure if this will overwrite the one defined above.

**Review by HosameldinMohamed — CHANGES_REQUESTED** [review]

File: iRonCub-Mk3/CMakeLists.txt (hunk 1 above)
Comment (HosameldinMohamed) [reply iRonCub-Mk3/CMakeLists.txt]: Something like `set(robot_list iRonCub-Mk3 iRonCub-Mk3-FT-ARMS iRonCub-Mk3-WALKING-TURBINES)`

Comment (davidegorbani) [reply iRonCub-Mk3/CMakeLists.txt]: Because I made a mistake; now I added the installation also for `iRonCub-Mk3-WALKING-NO-TURBINES` as you suggested.

**Review by HosameldinMohamed — APPROVED** [review]
Nice!

**Comment by davidegorbani** [comment]
Thank you!

**Pull request merged**